src/app/components/work-item-quick-add/work-item-quick-add.component.ts

```diff
@@ -105,7 +105,6 @@ export class WorkItemQuickAddComponent implements OnInit, OnDestroy, OnChanges,
     this.workItem.attributes = new Map<string, string | number>();
     this.workItem.relationships = new WorkItemRelations();
     this.workItem.type = 'workitems';
-    this.workItem.id = '42';
```

Comment:
the problem here is that `42` is not a valid `uuid` and furthermore, it does not make sense to provide a value for this field in the creation request. I can understand it was a work-around for an old bug, but it should not be kept any longer.